toast-tk-web-agent/src/main/java/io/toast/tk/agent/ui/utils/PanelHelper.java

```diff
@@ -0,0 +1,112 @@
+package io.toast.tk.agent.ui.utils;
+
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.Image;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+
+import javax.imageio.ImageIO;
+import javax.swing.BorderFactory;
+import javax.swing.Box;
+import javax.swing.BoxLayout;
+import javax.swing.ImageIcon;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.border.TitledBorder;
+import java.awt.Component;
+
+/**
+ * Configuration panel
+ */
+public class PanelHelper {
+	
+	private static String fontName = "Verdana";
+	public static Font FONT_TEXT_BOLD = new Font("SansSerif",Font.BOLD + Font.ITALIC,14);
+	public static Font FONT_TEXT_ITALIC = new Font(fontName,Font.ITALIC,14);
+	public static Font FONT_TEXT = new Font(fontName,Font.PLAIN,14);
+	public static Font FONT_TITLE_3 = new Font(fontName,Font.BOLD,16);
+	public static Font FONT_TITLE_2 = new Font(fontName,Font.BOLD,18);
+	public static Font FONT_TITLE_1 = new Font(fontName,Font.BOLD,22);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this field "FONT_TITLE_1" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3008)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this "public static FONT_TITLE_1" field final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1444)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make FONT_TITLE_1 a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)